### DISCUSSION TITLE
Android builds for rust libraries

### DISCUSSION THREAD

**Discussion by budchirp** [tailwindlabs/tailwindcss#13109] [Ideas]
Trying out the new tailwind v4 alpha3 on android gives this error:

![Screenshot_20240306_210649_Chrome Dev](https://github.com/tailwindlabs/tailwindcss/assets/80613396/95222a40-9db4-4070-8a65-f70702a7a1cd)

I think its caused because of the rust libraries is not compiled for Android.

**Comment by KrisBraun** [comment]
Thanks for [the PR](https://github.com/tailwindlabs/tailwindcss/pull/13115)! It's merged and will be in the next release.
